frontend/src/App.js

```diff
@@ -9,6 +9,9 @@ const theme = createMuiTheme({
   palette: {
     primary: { main: "#086375", contrastText: "#ffffff" },
     secondary: { main: "#119da4", contrastText: "#ffffff" },
+    background: {
+      default: "#F2FCFF",
```

Comment:
@hhummel is this a you change or a @robb33 change?  I dont know too much about changing the overall css theme stuff using Material-UI, but seems like a strange thing to have on this branch.

Reply:
The change in seed is a debugging flag i forgot to turn off when I did the Insurers PR, so I did it here.  I only affects the output when seed is run. The context in queue/views.py was to correct an error I was receiving.  The change in App.js in in response to Marcel's request that the background color be added to the theme.

Reply:
ok sounds good to me